airbyte-integrations/connectors/source-shipstation/README.md

```diff
@@ -0,0 +1,132 @@
+# Shipstation Source
+
+This is the repository for the Shipstation source connector, written in Python.
+For information about how to use this connector within Airbyte, see [the documentation](https://docs.airbyte.io/integrations/sources/shipstation).
```

Comment:
We should also add a direct documentation link of Ship Station as well since this airbyte-hosted link will have nothing for now.